modules/install/pages/install.adoc

```diff
@@ -4,28 +4,45 @@ Brief blurb about how JBoss EAP offers multiple ways to install...
 
 == Pre-requisites
 
-Talk about what pre-requisites are needed:
+* You have created an account on the Red Hat Customer Portal to download JBoss EAP files.
+* You have installed a https://access.redhat.com/articles/6961381[Red Hat supported OpenJDK 17]
+* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin)
+* Recommended to download all installation file under /opt partition. Ensure there is enough space under /opt partition (minimum 800MB).
+You have reviewed JBoss EAP 8.0 supported configurations.
+* You have access to the internet.
 
-* Red Hat supported OpenJDK 17 or higher (21 recommended?)
-* Recommended to create a separate *jboss* user account with no shell access (/sbin/nologin) 
-* Recommended to install under /opt. Can be a dedicated partition. Ensure there is enough space (suggest a good minimum number)
+== Red Hat JBoss Enterprise Application Platform 8 Installation Methods
+
+You can install JBoss EAP 8.0 by using one of the following installation methods:
+
+* *JBoss EAP using the Zip file*
+* *JBoss EAP using the JAR installer*
+* *JBoss EAP using archive installation method*
+* *JBoss EAP by using the RPM installation method*
 
 == Installing JBoss EAP using the Zip file
 
-Brief overview only. Details in lab section below....
+* This method can also be called jboss-eap-installation-manager installation method.
```

Comment:
Are you sure about this? is "jboss-eap-installation-manager" an official term used in the docs?